lib/github_api/client/projects/columns.rb

```diff
@@ -3,10 +3,15 @@
 
 module Github
   class Client::Projects::Columns < API
-    PREVIEW_MEDIA = "application/vnd.github.inertia-preview+json" # :nodoc:
     REQUIRED_COLUMN_PARAMS = %w(name).freeze
     REQUIRED_MOVE_COLUMN_PARAMS = %w(position).freeze
 
+    require_all 'github_api/client/projects/columns',
+                'cards'
+
+    # Access to Projects::Cards API
+    namespace :cards
```

Comment:
What I could do is, leave everything that requires a `column_id` nested (like: `api.projects.columns.cards.list 123` and un-nest anything that doesn't require reference to a column (e.g. `api.projects.cards.get 123`).

What do you think?